src/pages/api/ecodev-grants.ts

```diff
@@ -108,7 +109,10 @@ async function handler(req: NextApiRequest, res: NextApiResponse): Promise<void>
           // Document upload
           conn.sobject('ContentVersion').create(
             {
-              Title: `[Document] ${application.Project_Name__c} - ${createdLeadID}`,
+              Title: `[Document] ${truncateString(
+                (application.Project_Name__c || '').toString(),
```

Comment:
For some reason, in this case, the type of `Project_Name__c` is `string[]`, that is why I'm doing a `toString`

Reply:
shouldn't be plain `string`?

Reply:
Correct. This is coming from the `formidable` field types.